front/src/modules/infra/components/InfraSelector/InfraSelectorModalBodyEdition.tsx

```diff
@@ -152,7 +148,7 @@ const InfraSelectorModalBodyEdition = ({
                   <span className="ml-2" title={selectedFile.name}>
                     {selectedFile.name}
                   </span>
-                  <input type="file" onChange={handleSelect} accept=".json" />
+                  <input type="file" onChange={handleSelect} accept=".json,.railjson" />
```

Comment:
I think the [accept](https://developer.mozilla.org/fr/docs/Web/HTML/Reference/Attributes/accept) is only used to help the navigator to display relevant files to upload (when I open my files explorer, only the .json and .railjson files are proposed), but it does not validate the uploaded files.
We should keep the validation in the `validateFile` function if we want to restrict the import to railjson and json files